pkgs/tools/typesetting/xmlto/default.nix

```diff
@@ -48,18 +48,14 @@ stdenv.mkDerivation (finalAttrs: {
     done
   '';
 
+  strictDeps = true;
+
   # `libxml2' provides `xmllint', needed at build-time and run-time.
   # `libxslt' provides `xsltproc', used by `xmlto' at run-time.
   nativeBuildInputs = [
     autoreconfHook
     makeWrapper
-    flex
     getopt
-  ];
-
-  buildInputs = [
```

Comment:
Did you verify that XSLTPROC_PATH is still set in result/bin/.xmlto-wrapped?

Reply:
It is still set
```
LOCALE_PATH=/nix/store/lbyss088w6gi0ldkij6q1ik40p8ylqzw-glibc-2.40-36-bin/bin/locale
XMLLINT_PATH=/nix/store/wakgci6jbfk0klafr4lycn6b33f4i5f2-libxml2-2.13.5-bin/bin/xmllint
XSLTPROC_PATH=/nix/store/f18phs2dwhpzirdln64ki9k07x4m1cmw-libxslt-1.1.42-bin/bin/xsltproc
```